system/jcstress/build.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+
+<project name="JCStressTest" basedir=".">
+	<property name="LIB" value="jcstress" />
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml" />
+</project>
```

Comment:
Sorry, the above is not sufficient. We need the default targets.

```
<project name="JCStressTest" default="build" basedir=".">
	<taskdef resource="net/sf/antcontrib/antlib.xml" />
	<description>JCStress Test</description>

	<!-- Set global properties for this build -->
	<property name="DEST" value="${BUILD_ROOT}/system/jcstress" />
	<property name="SRC" location="." />

	<property name="LIB" value="jcstress" />
	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml" />

	<target name="init">
		<mkdir dir="${DEST}" />
	</target>
	
	<target name="dist" depends="init" description="generate the distribution">
		<copy todir="${DEST}">
			<fileset dir="${SRC}"/>
		</copy>
	</target>

	<target name="build">
		<antcall target="dist" inheritall="true" />
	</target>
</project>
```

Reply:
This should be tested via `BUILD_LIST=system/jcstress`

Reply:
Added (tested: Grinder/23946). Please check.

Reply:
Here's the test with `BUILD_LIST=system/jcstress` : Grinder/24008.